app/experimenter/kinto/README.md

```diff
@@ -0,0 +1,158 @@
+# Remote Settings (Kinto) Integration
```

Comment:
I think it could be worth linking to this README on the front-page like, "for detailed information on RS Kinto Integration see our docs"